forza.py

```diff
@@ -96,23 +96,23 @@ def analyze(self, performance_profile: bool=True):
             logger.debug(f'{self.analyze.__name__} started')
             self.shift_point = gear_helper.calculate_optimal_shift_point(self)
 
-            if performance_profile:
-                fig, ax = plt.subplots(2, 2)
+            # if performance_profile:
+            #     fig, ax = plt.subplots(2, 2)
 
-                # # gear vs ratio at 0, 0
-                helper.plot_gear_ratio(self, ax, 0, 0)
+            #     # # gear vs ratio at 0, 0
+            #     helper.plot_gear_ratio(self, ax, 0, 0)
+            #     # torque vs rpm at 0, 1
```

Comment:
Don't include test behaviors, like comment code in the PR. If you want to delete them, please show the reasons.